test.hxml

```diff
@@ -11,4 +11,4 @@
 --remap flash:openfl
 
 ## Neko
--cmd neko build/neko_test.n
\ No newline at end of file
+-neko build/neko_test.n
```

Comment:
Sorry, I'm still not clear on this. Can you please comment on something:

- If I just add `-cmd neko build/neko_test.n` to the end of `test.hxml`, I get an error: `HaxeWrapper.hx:73: /bin/sh: 1: neko build/neko_test.n: not found`
- If I put the command in quotes (`-cmd "neko build/neko_test.n"`), I get the tests running twice

What is the correct/expected behaviour? Should I put the command in quotation marks, and remove `-neko ...`?

Or, is this working differently for me than what you expect?